### PR TITLE
Misc/load test

### PR DESCRIPTION
**Pull request by boonwj**
This PR brings in load testing for the UI and Accessor components.

Testing is done using `https://locust.io/` and the tests works with the assumption that the backend services (ehr and holostorage) are loaded with FHIR data from `holorepo_demo' (#60)

The tests currently only cover the GET/PUT endpoints. The POST endpoint will generate many unique ids which will be hard to clean up.

### CODE REVIEW

File: Misc/tools/locust_load_test/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# LocustIO Load Test
```
Comment (nielsboecker) [inline Misc/tools/locust_load_test/README.md:1]: This README could be more specific about what UI means (client? server? both?)

File: Misc/tools/locust_load_test/locustfile-ui.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from locust import HttpLocust, TaskSet, task
+
+
+class UserBehavior(TaskSet):
+    @task
+    def getPatients(self):
+        self.client.get("/api/v1/patients")
```
Comment (nielsboecker) [inline Misc/tools/locust_load_test/locustfile-ui.py:7]: ~~Maybe api prefix could be extracted to a variable~~ Actually i kinda like it like it is

File: Misc/tools/locust_load_test/locustfile-ui.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    def getPractitioner(self):
+        self.client.get("/api/v1/practitioners/a100")
+
+    @task
+    def getImagingStudies(self):
+        self.client.get("/api/v1/imagingStudies")
+
+    @task
+    def getImagingStudiesOfPatients(self):
+        self.client.get("/api/v1/imagingStudies?pids=p100,p102")
```
Comment (nielsboecker) [inline Misc/tools/locust_load_test/locustfile-ui.py:27]: 😸

**Review by nielsboecker — APPROVED** [review]
Looks great! Would you also like to quickly summarise how it performs in OneNote, and/or write a section in the report on this?

File: Misc/tools/locust_load_test/README.md (hunk 1 above)
Comment (boonwj) [reply Misc/tools/locust_load_test/README.md:1]: I am really just hitting the server endpoints. So I will just change it to UI-Server

**Pull request merged**